src/power-bi/Shared.Dataset/definition/expressions.tmdl

```diff
@@ -7,35 +7,35 @@ expression 'Hub Storage URL' = "https://ftkmicfland5pd7gdo37oozc.dfs.core.window
 
 	annotation PBI_NavigationStepName = Navigation
 
-expression RangeStart = null meta [IsParameterQuery=true, Type="DateTime", IsParameterQueryRequired=false]
+expression RangeStart = #datetime(2024, 9, 1, 0, 0, 0) meta [IsParameterQuery=true, Type="DateTime", IsParameterQueryRequired=false]
 	lineageTag: 90085870-f708-4bc2-90c1-3a343e88a26b
 	queryGroup: '🛠️ Setup'
 
-	annotation PBI_ResultType = Null
+	annotation PBI_ResultType = DateTime
 
 	annotation PBI_NavigationStepName = Navigation
 
-expression RangeEnd = null meta [IsParameterQuery=true, Type="DateTime", IsParameterQueryRequired=false]
+expression RangeEnd = #datetime(2024, 9, 2, 0, 0, 0) meta [IsParameterQuery=true, Type="DateTime", IsParameterQueryRequired=false]
 	lineageTag: 695f1d3b-7b1a-49fa-8d83-f2c25dc787de
 	queryGroup: '🛠️ Setup'
 
-	annotation PBI_ResultType = Null
+	annotation PBI_ResultType = DateTime
 
 	annotation PBI_NavigationStepName = Navigation
 
 /// Loads data from the configured storage account for FinOps hubs or with Cost Management exports.
 /// 
 /// Learn more about FinOps hubs @ https://aka.ms/finops/hubs
 expression ftk_Storage = ```
-		(optional datasetType as text) =>
+		(optional datasetType as text, optional startDate as datetime, optional endDate as datetime) =>
```

Comment:
Do we care to take this as an input for a specific reason? Given the process for troubleshooting errors in functions, I was thinking it might be simplest to remove unnecessary parameters. I'm not holistically against it, tho.